devicedevex_dockerfiles/public/iot-device-cube-1.0.1/azure-c-app-20190411-ubuntu-arm64/compile_app.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -x
+
+application_name="iot_application"
+azure_c_sdk_directory="/work/azure-iot-sdk-c"
+cmake_directory="${azure_c_sdk_directory}/cmake"
+CMakeLists_file_path="${azure_c_sdk_directory}/CMakeLists.txt"
+CMakeLists_defaulit_file_path="${azure_c_sdk_directory}/default_CMakeLists.txt"
+output_folder="/work/output"
+
+if [[ $# == 0 ]]; then
+  echo "Usage: $0 <srcFileFolder> [application name]"
+  echo "default application name is iot_application"
```

Comment:
can be use ${application_name} instead of hard code.
echo "default application name is ${application_name}"

Reply:
Fixed.